lib/workers/master.js

```diff
@@ -111,8 +111,7 @@ class Master extends EventEmitter {
 
   /**
    * Emit an event on the worker side.
-   * @param {String} event
-   * @param {...Object} arg
+   * @param {...Object} items
```

Comment:
not sure Object is correct here, same for `sendEvent()`

Reply:
What should it be then?